websites/R/Roblox/presence.ts

```diff
@@ -460,201 +419,291 @@ presence.on("UpdateData", async () => {
-				presenceData.state = `Replying To ${
-					document
-						.querySelector(".composer-action-reply")
-						.querySelector(".user-link").textContent
-				}`;
-			} else if (document.querySelector(".keyboard-shortcuts-modal"))
-				presenceData.state = "Browsing Keyboard Shortcuts";
-			else if (document.querySelector(".flag-modal.in")) {
-				presenceData.state = "Flagging a Post";
-				delete presenceData.buttons;
-			} else if (document.querySelector(".composer-action-edit")) {
-				presenceData.state = "Editing a Post";
-				delete presenceData.buttons;
-			}
 			break;
 		}
 
-		case "talent.roblox.com": {
-			presenceData.details = "Surfing the Talent Hub";
-			presenceData.largeImageKey = dfLgImage;
-
-			if (pathname.includes("/search")) {
-				const searchResult = new URL(href).searchParams.get("query");
-				if (pathname.includes("/creators")) {
-					if (searchResult)
-						presenceData.state = `Searching for Creator: ${searchResult}`;
-					else presenceData.state = "Browsing Creators";
-				} else if (pathname.includes("/jobs")) {
-					if (searchResult)
-						presenceData.state = `Searching for Job: ${searchResult}`;
-					else presenceData.state = "Browsing Jobs";
-				}
-			} else if (
-				pathname.includes("/creators/") &&
-				!pathname.includes("/search/") &&
-				!pathname.includes("/settings")
-			) {
-				const Id = pathname.split("/")[2];
-				if (talentUserData[0] !== Id) {
-					talentUserData = [
-						Id,
-						document.head.title.replace("'s Creator Page - Talent Hub", ""),
+		case "create.roblox.com": {
+			presenceData.largeImageKey = Assets.CreateLogo;
+			const search = document.querySelector('[id="search-text-field"]');
+			switch (true) {
+				case pathname === "/landing": {
+					presenceData.details = "Browsing on the landing page";
+					break;
+				}
+				case pathname === "/": {
+					presenceData.details = "Browsing on the homepage";
+					break;
+				}
+				case pathname.includes("/dashboard/creations"): {
+					presenceData.details = "Creation's dashboard";
+					presenceData.state = `Tab: ${
+						document.querySelector('button[aria-selected="true"]')?.textContent
+					}`;
+					break;
+				}
+				case pathname.includes("analytics"): {
+					presenceData.details = "Viewing analytics";
+					presenceData.state = `Tab: ${
+						document.querySelector('button[aria-selected="true"]')?.textContent
+					}`;
+					break;
+				}
+				case pathname.includes("/translator-portal"): {
+					presenceData.details = "Browsing trough the translator portal";
+					break;
+				}
+				case pathname.includes("credentials"): {
+					presenceData.details = "Viewing the credentails manager";
+					break;
+				}
+				case pathname.includes("/docs"): {
+					switch (true) {
+						case !!search: {
+							presenceData.details = search?.getAttribute("value")
+								? "Searching for:"
+								: "Searching...";
+							presenceData.state = document
+								.querySelector('[id="search-text-field"]')
```

Comment:
For consistency with other id query selectors
```suggestion
								.querySelector("#search-text-field")
```